test/config/mocks/i18n-loader.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('esn.i18n')
+  .factory('esnI18nLoader', esnI18nLoader);
+
+function esnI18nLoader($q) {
+  var getCatalogsPromise;
+  return function(options) {
+    if (!getCatalogsPromise) {
+      getCatalogsPromise = getCatalogs();
+    }
+
+    return getCatalogsPromise
+      .then(function(catalogs) {
+        if (!catalogs[options.key]) {
+          return $q.reject(Error('No catalog found for ' + options.key));
```

Comment:
You use both `$q` and `Promise` in this file, why not `Promise` here?

Reply:
Sorry, I didn't notice this part of code, but only `$q` should be used here.

Reply:
Fixed